### ISSUE TITLE
Some setting descriptions remain untranslated

### ISSUE THREAD

**Issue by walterl** [metabase/metabase#10770]
Settings now use the `deferred-trs`/`deferred-tru` macros for their descriptions. Some settings with long descriptions break those descriptions up into lines by using

```clojure
(str (deferred-trs "long text line 1")
     (deferred-trs "long text line 2")
     ; ...
)
```

The `str` has the effect of immediately looking up the translations for the wrapped strings.

Since settings descriptions are evaluated at compile-time, before locales are loaded, those strings remain untranslated (English US). The translation lookups of these strings should be deferred, like all other translations using `deferred-tr*`, until they are needed (after compile time).

## Option 1
Add a multi-arity version of the `deferred-tr*` macros, which allows for the concatenation of multiple translated strings, after the deferred lookup was done.

This is the simpler and easier option, but leaves the l10n issue described below.

## Option 2
Join the strings into single, long strings, including appropriate line breaks. This is slightly more localizer-friendly, since it allows translators to move line endings in their translations, to account for varying length of translated lines.

The downside is that this would require some extra processing on existing translations, either by each translation team, or we could apply some action like joining the lines automatically.

**Comment by camsaul** [comment]
In #12558 I came up with a few more ways we can address this.

**Comment by qnkhuat** [comment]
I'm looking through this and I think this should be closed in favor #12558(duplication of this issue) has been closed.

**Comment by flamber** [comment]
@qnkhuat 
You are asking for this to be closed in favor of #12558, which is already closed as a duplicate of this issue?
That doesn't make sense. We need to keep an issue open until it's fixed.
